app/views/rooms/_room_icecomm_helper_functions.html.erb

```diff
@@ -1,11 +1,19 @@
-var comm = new Icecomm('9gv3ODZorBS2kVuNwnugBeV432PcNX3Qn0HmW33ofxK/u5Xolu');
+var comm = new Icecomm('9gv3ODZorBS2kVuNwnugBeV432PcNX3Qn0HmW33ofxK/u5Xolu', {debug: true});
 comm.connect(window.location.pathname);
 
-    // Set up the WebSocket in onload; so that it is available for use elsewhere.
-    var host = "localhost";
-    var port = "8888";
-    var uri = "/recognize";
-    ws = new WebSocket("ws://" + host + ":" + port + uri );
+    // Set up the WebSocket
```

Comment:
Open secure websocket (wss) to Python server if we are connecting through https. If we are not connecting through https that means we are testing on localhost, so we should instead open a regular websocket (ws).